src/ctim/document.clj

```diff
@@ -63,6 +64,7 @@
            ["structures/sighting.md" s/Sighting ->markdown]
            ["json/sighting.json" s/Sighting ->json]
            ["structures/tool.md" tool/Tool ->markdown]
+           ["json/target_record.json" target-record/TargetRecord ->json]
```

Comment:
You put it between the tool doc generation. You forgot the markdown generation 
```suggestion
           ["structures/target_record.md" target-record/TargetRecord ->markdown]]
           ["json/target_record.json" target-record/TargetRecord ->json
           ["structures/tool.md" tool/Tool ->markdown]]
```
BTW I missed in previous PRs that you forgot the markdown.

Reply:
added in 78cfb65195e849f4042b7da0d95e8c137dfd2de8